src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -299,173 +425,238 @@ struct BlueprintParticleMesh
     }
 
     // set the default connectivity
-    sidre::Array<int> arr(m_topoGroup->createView("elements/connectivity"), SZ, SZ);
+    // Maybe be required by an old version of visit.  May not be needed by newer versions of visit.
+    sidre::Array<int> arr(
+      m_topoGroups[domainIdx]->createView("elements/connectivity"),
+      SZ,
+      SZ);
     for(int i = 0; i < SZ; ++i)
     {
       arr[i] = i;
     }
   }
 
+  template <int NDIMS>
+  axom::Array<primal::Point<double, NDIMS>> getPoints(int domainIdx)
+  {
+    // This code is untested and currently unused.
```

Comment:
Any reason to keep this function if it's untested and unused?

Reply:
That's an obsolete comment.  Removing.  Thanks.